snap/snapcraft.yaml

```
@@ -237,6 +200,8 @@ apps:
       - desktop
       - desktop-legacy
       - screen-inhibit-control
+      - x11
+      - wayland
```

I got curious about this (I thought the `desktop` and `desktop-legacy` plugs implied `wayland` and `x11`, but apparently it's more complex than that). Looking at [the docs](https://snapcraft.io/docs/desktop-interfaces) there seems to be a recommended list of plugs for electron apps.

From that list we're missing:

```
      - alsa
      - avahi-observe
      - cups-control
      - gsettings
      - upower-observe
```

Combining that with what I believe Signal *actually* uses, would this be appropriate?

```suggestion
      - x11
      - wayland
      - alsa
      - upower-observe
```

---

I'm not inclined to add any more unless things aren't working. Going to leave this for now.